src/java.base/share/classes/java/lang/ModuleLayer.java

```diff
@@ -881,6 +881,19 @@ public Optional<Module> findModule(String name) {
                 .findAny();
     }
 
+    /**
+     * Returns the module with the given name in this layer only.
+     *
+     * @param  name
+     *         The name of the module to find
+     *
+     * @return The module with the given name or {@code null}
+     *         if there isn't a module with this name in this layer
+     */
+    Module findModuleInThisLayer(String name) {
+        return nameToModule.get(name);
+    }
+
```

Comment:
What would you think about replacing this with addEnableNativeAccess(String name) so it can be called by JLA. addEnableNativeAccess. The reason is that the JLA methods are usually just calls to some non-public method but the changes mean mean there is "core" in the JLA method that is not easy to find.

Reply:
I've tried to that here:
https://github.com/openjdk/jdk/pull/18106/commits/e17cd3722724cbc6aa298f7b789c6574554af6ea